prov/verbs/src/verbs_domain.c

```diff
@@ -200,6 +229,14 @@ static struct fi_ops fi_ibv_fid_ops = {
 	.ops_open = fi_no_ops_open,
 };
 
+static struct fi_ops fi_ibv_rdm_fid_ops = {
+	.size = sizeof(struct fi_ops),
+	.close = fi_ibv_domain_close,
+	.bind = fi_ibv_rdm_domain_bind,
```

Comment:
This could be added to fi_ibv_fid_ops itself and there is no need to define separate rdm specific ops. EQ events like FI_MR_COMPLETE would be useful to all endpoint types.